zulipterminal/model.py

```diff
@@ -957,12 +973,10 @@ def _handle_typing_event(self, event: Event) -> None:
                     and event['sender']['email'] in narrow[0][1].split(',')):
                 if event['op'] == 'start':
                     user = self.user_dict[event['sender']['email']]
-                    self.controller.view.set_footer_text([
-                        ' ',
-                        ('code', user['full_name']),
-                        ' is typing...'
-                    ])
+                    self.is_typing = True
+                    self.animate_typing(user)
```

Comment:
I am not sure about the name `user` here. This person is actually a recipient and the ZT user is the _user_.

Reply:
This was how it originally was, though I think `sender` maybe a better word here?

Reply:
Hmm, `sender` would be a bit ambiguous as in this case both, the user and the recipient, could be perceived as a sender. I believe `recipient` would be apt.